src/operations.cc

```diff
@@ -221,10 +221,8 @@ namespace sharp {
 
     VipsImage *vips_matrix = kernel.get_image();
 
-    int i = 0;
-    for(int k : kernel_v) {
```

Comment:
I was trying to fix the mysterious appveyor failure. I suppose this has nothing to do with it. I can go back on this commit.